packages/types/src/codec/Compact.ts

```diff
@@ -93,11 +111,35 @@ export default class Compact extends UInt {
     );
   }
 
+  get bitLength (): UIntBitLength {
+    return this.raw.bitLength;
+  }
+
   get encodedLength (): number {
     return this.toU8a().length;
   }
 
+  toBn (): any {
```

Comment:
Not crazy about this either... we don't have all the `.add` stuff from the `Uint` base. Maybe the `Uint extends BN` could mangle this one as well to be a bit more user-friendly?

Reply:
... the above is a slight issue since we can either Compact UInt or Moment now.